apps/storefront/src/pages/Dashboard/index.tsx

```diff
@@ -52,6 +56,8 @@ function Dashboard(props: PageProps) {
   const b3Lang = useB3Lang();
 
   const [isRequestLoading, setIsRequestLoading] = useState(false);
+  const [tempCompanyId, setTempCompanyId] = useState(1);
+  const [cartEntityId, setCartEntityId] = useState<string>('');
```

Comment:
reading `cartId` in both places removes the need for `setCartEntityId`, we do have to configur `tempCompanyId` here. Can we give it a more explicit name:


```tsx
const [confirmMasquerade, setConfirmMasquerade] = useState<{ companyId: number } | undefined>(undefined);
```